.github/workflows/test.yml

```diff
@@ -187,6 +187,7 @@ jobs:
     if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.base_ref == 'main' || github.base_ref == 'cli-2.0') || github.event_name == 'merge_group'
     needs: [yarn-build]
     strategy:
+      fail-fast: false
```

Comment:
I do think that the cosmos e2e test should be required in CI, just so we don't accidentally break deploy tooling or agent support

Reply:
I think we should still have `fail-fast: false` to let both tests run til completion, but promote `cosmwasm` to a required check in the repo settings